filter-effects-2/Overview.bs

```diff
@@ -50,28 +54,281 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
+performance breakdown. One set of nested backdrop-filtered elements would
+require four (4) times the CPU/GPU and memory. Two nested sets would require
+eight (8) times the CPU and memory. Clearly, this is not a scalable approach.
+
+Note: this definition encompases fewer element types than the definition for a
+Stacking Context. In particular, a Backdrop Root is not formed by elements with
+z-index applied, fixed or sticky-positioned elements, and elements with
+transforms applied. This allows elements with backdrop-filter or mix-blend-mode
+to apply to elements higher up the DOM tree than would otherwise apply if they
+stopped at the parent Stacking Context. For example, a container can be used to
+contain elements with backdrop-filter applied, and that container can use
+transforms for animation or positioning, while still allowing the
+backdrop-filter to apply to the background behind the container.
+
+<div class="example">
+<p>Given the following html code:</p>
+<pre><code highlight=html>
+&lt;html style="background:lightgrey;"&gt;
+  &lt;div class="box"&gt;
+    &lt;b&gt;"Box"&lt;/b&gt;&lt;br&gt;&lt;br&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+    velit esse cillum
+    &lt;div class="dialog"&gt;&lt;/div&gt;
+  &lt;/div&gt;
+&lt;/html&gt;
+</code></pre>
+<p>And the following style rules:</p>
+<pre><code highlight=css>
+&lt;style&gt;
+  div {
+    position: absolute;
+    width: 200px;
+    height: 200px;
+    top: 70px;
+    left: 70px;
+  }
+  .box {
+    background: white;
+    filter: invert(1);
+  }
+  .dialog {
+    border: 10px solid blue;
+    backdrop-filter: blur(2px);
+  }
+&lt;/style&gt;
+</code></pre>
+<p>Consider the sequence of painting operations to produce the final result,
+   assuming that the "dialog" element accesses <b>everything behind it</b>, all
+   the way to the root element. <b>Important note:</b> this is <b>not</b> how
+   the Backdrop Root is defined - this example is intended to illustrate why a
+   Backdrop Root is formed by elements containing filters.</p>
+<div class="figure">
+  <img alt="ALT TEXT HERE" src="examples/step1.png" style="width:30%"/>
+  <p class="caption"><b>Step 1.</b> "Box" is rendered, but filters are not yet applied.</p>
+  <img alt="ALT TEXT HERE" src="examples/step2.png" style="width:30%"/>
+  <p class="caption"><b>Step 2.</b> "Dialog" is about to be rendered, but it has
+    backdrop-filter applied. Since, for this example, the backdrop-filter "sees"
+    all the way to the root, the "Box" element needs to be temporarily
+    "completed" so that the final product can be used as input to "Dialog". This
+    means applying it's "filter: invert(1)" style. Note, this is where the
```

Comment:
it's -> its

Reply:
Thanks! That's one of my pet peeves, I hate that I left it there.